lib/AFP/Data/Word16.hs

```diff
@@ -0,0 +1,309 @@
+{-# LANGUAGE CPP #-}
+
+-- | Word16 module AFP.to be used with ShortByteString.
+-- All function assumes that 'Word16' is encoded in UTF16-LE.
+--
+-- Works with Unicode7+ (GHC 7.10+ / Base library 4.8+).
+module AFP.Data.Word16 (
```

Comment:
Similar to https://hackage.haskell.org/package/word8-0.1.3/docs/Data-Word8.html

Reply:
This needs to be converted to LE though, same for `AFP.Data.ByteString.Short.Word16`.